docs/source/sophronia.rst

```diff
@@ -0,0 +1,251 @@
+Sophronia
+=========
+
+*From ancient Greek: sensible, prudent.*
+
+
+This city processes each S2 signal previously selected as pmaps in
+irene assuming a unique S1 within an event to produce a set of
+reconstructed energy depositions (**Hits**). Hits consist of three
+dimensional coordinates with an associated energy (PMT signal) and
+charge (SiPM signal). The city contains a peak/event filter, which can
+be configured to find events with a certain number of S1/S2 signals
+that satisfy certain properties. Currently, the city is designed to
+accept only 1 S1 signal. If the configuration allows for more than one
+S1, the first one will be used to estimate z. Furthermore, hits can be
+optionally corrected for geometrical and lifetime effects. Besides
+hits, the city also stores the global (x, y) position of each S2
+signal.  The tasks performed are:
+    - Classify peaks according to the filter.
+    - Filter out events that do not satisfy the selector conditions.
+    - Resample S2 signals.
+    - Compute a set of hits for each slice in the resampled S2 signal.
+    - If there are more than one hit per slice, share the energy
+      according to the charge recorded in the tracking plane.
+    - If correction maps are provided, applies corrections to hits.
+
+.. _Sophronia input:
+
+Input
+-----
+
+ * ``/Run/events``
+ * ``/Run/runInfo``
+ * ``/PMAPS/S1``
+ * ``/PMAPS/S2``
+ * ``/PMAPS/S2Si``
+
+.. _Sophronia output:
+
+Output
+------
+
+  * ``/DST/Events``: summary of the S1 and S2 information of the reconstructed events. See :doc:`dorothea` for more details.
+  * ``/RECO/Events``: list of hit data for each event.
+  * ``/Filters/s12_selector``: flag for whether an event passed the S1 and S2 selections
+  * ``/Filters/valid_hit``: flag for whether an event passed the empty pmap filter
+
+.. _Sophronia config:
+
+Config
+------
+
+Besides the :ref:`Common arguments to every city`, *Sophronia* has the following arguments:
+
+.. list-table::
+   :widths: 50 40 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+
+   * - ``drift_v``
+     - ``float``
+     - Drift velocity. Used to calculate the z position from the drift time.
+
+   * - ``s1_params``
+     - ``dict``
+     - Selection criteria for S1 signals (see `Peak filtering`_)
+
+   * - ``s2_params``
+     - ``dict``
+     - Selection criteria for S2 signals (see `Peak filtering`_)
+
+   * - ``global_reco_algo``
+     - ``XYReco``
+     - Reconstruction algorithm applied to entire peaks (kDST)
+
+   * - ``global_reco_params``
+     - ``dict``
+     - Configuration parameters for the global recontruction algorithm
+
+   * - ``rebin``
+     - ``int``
+     - Specifies the level of resampling or grouping of consecutive samples for reconstruction. Must be :math:`\geq` 1
+
+   * - ``rebin_method``
+     - ``RebinMethod``
+     - Which resampling method to use (usually stride)
```

Comment:
Same as above. Wouldn't it be simpler if you say: "Resampling method being used (usually stride)" or just "Resampling method (usually stride)"?